fairscale/nn/data_parallel/fully_sharded_data_parallel.py

```diff
@@ -274,6 +274,9 @@ def __init__(
         force_input_to_fp32: bool = False,
         verbose: bool = False,
         cpu_offload: bool = False,
+        gradient_predivide_factor: Optional[float] = None,
```

Comment:
Do you need to sync to main?

We are moving away from adding params to the API. can you create a config object that is suitable for this keeping in mind other params that can be added as well?